### DISCUSSION TITLE
5월달 회고록

### DISCUSSION THREAD

**Discussion by shkisme** [BDD-CLUB/challenge-every-week-posting#533] [5. 회고]
https://shkisme.vercel.app/2024-05-log

이번 5월달을 되돌아보는 회고를 작성했습니다!

**Comment by 02ggang9** [comment]
요즘에는 기술 글보다 이런 일상 글이 너무 재미있네요~ ㅋㅋㅎ

**Reply by shkisme** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋ 재밌게 읽어주셔서 캄사합니닷 🤩
